internal/godoc/render.go

```diff
@@ -114,13 +114,20 @@ func (p *Package) DocPackage(innerPath string, modInfo *ModuleInfo) (_ *doc.Pack
 	}
 	var allGoFiles []*ast.File
 	for _, f := range p.Files {
-		allGoFiles = append(allGoFiles, f.AST)
+		if f.Name != "" {
+			allGoFiles = append(allGoFiles, f.AST)
+		}
 	}
+
+	FindOverloadFuncThenAdd(p)
```

Comment:
It looks like we have two implementations:
* Modify the process of ast.Package => doc.Package;
* Transform doc.Package
Personally, I prefer the latter. This way, you don’t need to modify other people’s function behavior. You only need to have an independent function and the coupling is lower.

Reply:
> It looks like we have two ways to implement:
> 
> * Modify the process of ast.Package => doc.Package;
> * Transform doc.Package
> 
> Personally, I prefer the latter. This way, you don’t need to modify other people’s function behavior. You only need to have an independent function and the coupling is lower.

ok